src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -720,6 +721,15 @@ Status ServiceBasedNodeInfoAccessor::AsyncReportHeartbeat(
   return Status::OK();
 }
 
+void ServiceBasedNodeInfoAccessor::AsyncReReportHeartbeat() {
+  if (cached_heartbeat_.has_heartbeat()) {
+    RAY_LOG(INFO) << "Rereport heartbeat.";
+    client_impl_->GetGcsRpcClient().ReportHeartbeat(
+        cached_heartbeat_,
+        [](const Status &status, const rpc::ReportHeartbeatReply &reply) {});
```

Comment:
[](const Status &status, const rpc::ReportHeartbeatReply &reply) {} can be instead of nullptr?

Reply:
No it can't, since the callback will directly invoked in rpc client without any check, see https://github.com/ray-project/ray/blob/master/src/ray/rpc/gcs_server/gcs_rpc_client.h#L61